rust/agents/relayer/Cargo.toml

```
@@ -33,6 +33,7 @@ hyperlane-base = { path = "../../hyperlane-base" }
 hyperlane-ethereum = { path = "../../chains/hyperlane-ethereum" }
 num-derive.workspace = true
 num-traits.workspace = true
+derive_more = "0.99.17"
```

If we specify the patch version it won't auto update for security/bug fixes.
```suggestion
derive_more = "0.99"
```